test/integration/src/main/java/io/pravega/test/integration/utils/PasswordAuthHandlerInput.java

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright (c) 2019 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.integration.utils;
+
+import com.google.common.base.Preconditions;
+import com.google.common.base.Strings;
+import lombok.Data;
+import lombok.Getter;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * This is a helper class for tests and may be used for generating the input file for the PasswordAuthHandler - the
+ * default AuthHandler implementation.
+ */
+public class PasswordAuthHandlerInput {
+
+    @Getter
+    private File file;
+
+    public PasswordAuthHandlerInput() {
+        this("auth_file", ".txt");
+    }
+
+    public PasswordAuthHandlerInput(String fileName, String extension) {
+        try {
+            file = File.createTempFile(fileName, extension);
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+    }
+
+    public void postEntry(Entry entry) {
+        postEntries(Arrays.asList(entry));
```

Comment:
It is good to have a precondition to validate the `entry` for not being null

Reply:
Sure. I've pushed a new commit that checks arguments for this and other public methods of the class.